cfgov/unprocessed/css/skip-nav.less

```diff
@@ -20,8 +20,13 @@
 }
 
 #skip-nav:focus {
```

Comment:
It's only a super button on focus because it's hidden the rest of the time. But what's if we made it a super button all the time that's hidden with positioning till focused similar to the slide out menu on small screens?